### ISSUE TITLE
[Suggestion] Survival/Creative features separation 

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#3119]
Increasingly, survival elements are being introduced to the game which is fantastic, but at this early stage some of them are restricting during a lot of play time. Things like character death, fast character mode and the ability to teleport between ships freely are examples of features that may be different between a survival and creative mode.

Given that the mode we have available to us at the moment is labelled Creative mode, I suggest enabling fast move mode and teleport to all by default on all Creative mode worlds, with options to disable them in the debug menu. Creative mode is ideally a mode where you can build and explore without limits, having to open up the debug menu and enable teleport to all ships and fast mode every time you boot the game for some building is a chore. These tools are used literally every session, and fast mode is something I and others interact with multiple times a minute. With the new GUI element that shows your speed multiplier, I don't think this will be confusing to players. Actually, I've met many players who didn't even know these essential tools existed as they're a little tricky to find.

Regarding the ability to die, it's great from a gameplay perspective but frustrating if you're playing around with weapons etc. Perhaps an F3 menu toggle for "Godmode" as it's called in other games where the character is invulnerable would help with this.

To be clear, I'm not hating on these features and the drive towards Survival, it's fantastic! I just think that Creative and Survival features need to be distinct in their own modes rather than bleeding into each other.

**Comment by tsunamayo** [comment]
Yeah it make sense, but some stuff would be harder to do. Typically how I should handle the case where people goes straight into the sun or the planet and the ship get destroyed if I cant kill the player? (newcomer seems to like that to do that a lot) Cheers

**Comment by ProPeach** [comment]
Yes that's a good point, my idea was that the "godmode" ability where you can't die would be there in the debug menu for more experienced players to use rather than on by default like the fast speed or teleport options would be

**Comment by Xerma** [comment]
I do agree that the teleport and fast keys options should be enabled by default in creative worlds. Each time I load in I without a doubt always turn those on. And while yes we need to be able to show and experience that player/ship death is a thing in the game it does seem strange to die in a creative world. For now I would like to see a godmode option to enable, or have enabled by default
